.devcontainer/devcontainer.json

```diff
@@ -2,10 +2,8 @@
 // https://github.com/microsoft/vscode-dev-containers/tree/v0.183.0/containers/python-3
 {
 	"name": "Ubuntu 20.04 Python 3",
-	"build": {
-		"dockerfile": "Dockerfile",
-		"context": ".."
-	},
+	"image": "public.ecr.aws/w5r9l1c8/swsoc-docker-lambda-base:latest",
```

Comment:
Please add a comment to link to the repository that defines this container so that a developer can look up what capabilities are provided.